lib/user.rb

```diff
@@ -0,0 +1,27 @@
+class User
+
+  attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << joke
+  end
+
+  def tell(jokester, joke)
+    jokester.learn(joke)
+  end
+
+  def joke_by_id(id)
+    @jokes.find do |joke|
+      joke.id == id
+    end
+  end
+
+  def repeated_jokes?
```

Comment:
I see you have this method laid out in both the `open_mic` class and this one-- we want it in the `open_mic` class only, because the interaction pattern tells us that it's being called on an instance of `OpenMic`.